cpp/src/barretenberg/stdlib/primitives/bit_array/bit_array.test.cpp

```diff
@@ -12,105 +26,109 @@ namespace {
 auto& engine = numeric::random::get_debug_engine();
 }
 
-typedef stdlib::bool_t<honk::StandardHonkComposer> bool_t;
-typedef stdlib::field_t<honk::StandardHonkComposer> field_t;
-typedef stdlib::uint32<honk::StandardHonkComposer> uint32;
-typedef stdlib::witness_t<honk::StandardHonkComposer> witness_t;
-typedef stdlib::bit_array<honk::StandardHonkComposer> bit_array;
-typedef stdlib::byte_array<honk::StandardHonkComposer> byte_array;
+template <class Composer> class BitArrayTest : public ::testing::Test {};
+
+using ComposerTypes =
+    ::testing::Types<honk::StandardHonkComposer, plonk::StandardComposer, plonk::TurboComposer, plonk::UltraComposer>;
+TYPED_TEST_SUITE(BitArrayTest, ComposerTypes);
 
-TEST(stdlib_bit_array, test_uint32_input_output_consistency)
+TYPED_TEST(BitArrayTest, test_uint32_input_output_consistency)
```

Comment:
Would you mind also CamelCase'ing the test name `test_uint32_input_output_consistency`?

Reply:
I see you do this in some places but not others.